analyzers/tests/ITs.JsonParser.Test/JsonParsingTest.cs

```diff
@@ -1,12 +1,187 @@
-﻿namespace SonarAnalyzer.Net8.Test;
+﻿namespace ITs.JsonParser.Test;
 
 [TestClass]
 public class JsonParsingTest
 {
-    //TODO: Add tests during implementation
+    public TestContext TestContext { get; set; }
+
     [TestMethod]
-    public void NewIssueReports_Test()
+    public void IssueParser_Execute_Single()
```

Comment:
Will not be needed after the class rename
```suggestion
    public void Execute_Single()
```

Reply:
I think `Execute_Single` is a bit hard to search for in the Test explorer, with the `IssueParser` prefix you can get all of them definitely.

Reply:
We generally don't duplicate the class name in the test name. The searching in VS Test Explorer is pretty weak anyway. Ctrl+E, S and grouping by class name does the job in most of the cases